third_party/build.sh

```diff
@@ -45,7 +47,7 @@ esac
 
 # Link everything
 cd ..
-ar rvs libroutingkit.a Client.o RoutingKit/build/* temp/*
+/opt/homebrew/opt/llvm@14/bin/llvm-ar rvs libroutingkit.a Client.o RoutingKit/build/* temp/*
```

Comment:
This will fail on linux. Maybe set the alias from above conditional on the `GOOS`.